site/package.json

```diff
@@ -136,8 +134,7 @@
     "typescript": "4.8.2",
     "webpack": "5.74.0",
     "webpack-bundle-analyzer": "4.6.1",
```

Comment:
Do we still need these?

Reply:
It is kinda used to analyze the build, but I don't think we use this anywhere I will remove this after just for caution.